tests/utils/test_util.py

```diff
@@ -274,3 +274,93 @@ def test_is_gzip_n(self):
         self.assertFalse(Util.is_gzip(b'\x1f'))
         self.assertFalse(Util.is_gzip(b'\x1f\x8bxxx'))
         self.assertFalse(Util.is_gzip(b'\x1f\x8b\x02'))
+
+    def test_json_load_p(self):
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            self.assertTrue(os.path.isdir(tmp_dir))
+            file_path = os.path.join(tmp_dir, __name__)
+            with open(file_path, "wb") as f:
+                f.write(b'{}')
+            data = Util.json_load(file_path)
+            self.assertIsInstance(data, dict)
+
+            with open(file_path, "wb") as f:
+                f.write(b'[]')
+            data = Util.json_load(file_path)
+            self.assertIsInstance(data, list)
+
+            with open(file_path, "wb") as f:
+                f.write(b'"' + AZ_DATA + b'"')
+            data = Util.json_load(file_path)
+            self.assertIsInstance(data, str)
+            self.assertEqual(AZ_STRING, data)
+
+            rand_int = random.randint(-100, 100)
+            with open(file_path, "wb") as f:
+                f.write(str(rand_int).encode())
+            data = Util.json_load(file_path)
+            self.assertIsInstance(data, int)
+            self.assertEqual(rand_int, data)
+
+            rand_float = rand_int / 3.14
+            with open(file_path, "wb") as f:
+                f.write(str(rand_float).encode())
+            data = Util.json_load(file_path)
+            self.assertIsInstance(data, float)
+            self.assertEqual(rand_float, data)
+
+            with open(file_path, "wb") as f:
+                f.write(b'true')
+            data = Util.json_load(file_path)
+            self.assertIsInstance(data, bool)
+            self.assertTrue(data)
+
+            with open(file_path, "wb") as f:
+                f.write(b'null')
+            data = Util.json_load(file_path)
+            self.assertIsNone(data)
+
+    def test_json_load_n(self):
+        self.assertIsNone(Util.json_load("not_existed_path"))
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            self.assertTrue(os.path.isdir(tmp_dir))
+            file_path = os.path.join(tmp_dir, __name__)
+            with open(file_path, "wb") as f:
+                f.write(AZ_DATA)
+            self.assertIsNone(Util.json_load(file_path))
+
+    def test_json_dump_p(self):
```

Comment:
Imo it's a bit unclear what is being tested here
Docstring comments might be a nice idea
please conciser adding them with that or different PR